application.nix

```diff
@@ -175,18 +175,26 @@ rec {
       services.udev.extraRules = ''
         ACTION=="change", SUBSYSTEM=="drm", RUN+="${pkgs.systemd}/bin/systemctl start select-display.service"
       '';
-      systemd.services."select-display" = {
+      systemd.services."select-display" = let
+        PlayXauhtorityFile = "${config.users.users.play.home}/.Xauthority";
```

Comment:
Char swap, I guess file is still greppable because it appears in same line in actual string though.

Xauhtority/Xauthority